src/prisma/schema.prisma

```diff
@@ -38,3 +40,16 @@ model Profile {
   website                  Json?
   tdaGradYearBannerVisible Json
 }
+
+model Post {
+  id                 Int   @id @default(autoincrement())
+  userId             Int?   @unique
+  user               User?  @relation("UserToPost",fields: [userId], references: [id])
+  content            String
+  image              String?
+  likes              Int?      @default(0)
+  dislikes           Int?      @default(0)
+  isDraft            Boolean?
```

Comment:
This should be required also, and lets have it default to false. In the DTO it can become optional, so that unless the client says isDraft = true, then we will assume it is not a draft.